cli_tools/gce_ovf_export/exporter/instance_disks_exporter.go

```diff
@@ -155,10 +155,16 @@ func (ide *instanceDisksExporterImpl) addExportDisksSteps(w *daisy.Workflow, ins
 		if params.ComputeServiceAccount != "" {
 			varMap["compute_service_account"] = params.ComputeServiceAccount
 		}
+		p := path.Join(params.WorkflowDir, "/export/disk_export_ext.wf.json")
 		exportDiskStep.IncludeWorkflow = &daisy.IncludeWorkflow{
```

Comment:
this is  a bit anti intuitive - feels like it's initialzied twice. would you wrap it so it looks like a single operation?

Reply:
makes sense -- refactored into its own method.